main.tf

```diff
@@ -226,11 +226,13 @@ module "oke" {
   admission_controller_options                            = var.admission_controller_options
 
   # oke node pool parameters
-  node_pools            = var.node_pools
-  node_pool_name_prefix = var.node_pool_name_prefix
-  node_pool_image_id    = var.node_pool_image_id
-  node_pool_os          = var.node_pool_os
-  node_pool_os_version  = var.node_pool_os_version
+  node_pools                          = var.node_pools
+  node_pool_name_prefix               = var.node_pool_name_prefix
+  node_pool_image_id                  = var.node_pool_image_id
+  node_pool_os                        = var.node_pool_os
+  node_pool_os_version                = var.node_pool_os_version
+  is_pv_encryption_in_transit_enabled = var.is_pv_encryption_in_transit_enabled
```

Comment:
Please use the imperative style. See coding conventions: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/docs/codingconventions.adoc

Reply:
you can change this to `enable_pv_encryption_in_transit`